library/reals/README.md

```diff
@@ -0,0 +1,30 @@
+# Reals
+## Computable reals in Coalton
+
+This library is built on [computable-reals](https://github.com/stylewarning/computable-reals).
+
+### Creal 
+
+The primary type is a `Creal`, a wrapper on the computable-reals type of the same name.
+
+As stated in the computable-reals README:
+
+"Computable real numbers `x` are interpreted as (potentially) infinite fractions in base 2 that are specified through a rule for computation of an integer `a` with `|(2^k)*x - a| <= 1` for any `k>=0`."
+
+### Instances:
+
+`EQ` and `Ord` have been implemented in order to fulfill the needs of the `Num` typeclass, though they are not exact. Their definitions are shaky because they can only be verified up to an arbitrary, specified precision. The precision threshold for `Creal` comparison is stored in the lisp variable `*creal-comparison-threshold*`, which can be accessed using the library functions `creal-prec` and changed using `set-creal-prec`. The default precision is `106`.
```

Comment:
I wouldn't mention the Lisp variable name, and the Coalton function names need changing.